challenge_5/python/alexbotello/FindTheDifference.py

```diff
@@ -0,0 +1,21 @@
+from collections import Counter
+
+
+class Solution:
+    def findTheDifference(self, s, t):
+
+        s = dict(Counter(s))
+        t = dict(Counter(t))
+
+        for key in t.keys():
+            if key not in s.keys():
```

Comment:
This is O(n^2) as you have a nested for loop. Are you sure there isn't a way to avoid that? Try using [defaultdict](https://docs.python.org/3/library/collections.html#collections.defaultdict) instead of the standard `dict`.